SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| master  | Yes                |
+| >= 1.9.0| Yes                |
```

Comment:
So after 1.9.1 is released we still support 1.9.0?

Reply:
Well would think so, any security updates for 1.9.x would go into 1.9.y where y is a new number.